src/docs/development/platform-integration/web-images.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Displaying images on the web.
+short-title: Web Images
+description: Learn how to load and display images on the web.
+---
+
+The web supports the standard [Image][1] widget to display images. However,
+because web browsers are built to run untrusted code safely, there are certain
+limitations in what you can do with images compared to mobile and desktop
+platforms. This page explains these limitations and offers ways to work around
+them.
+
+# Background
+
```

Comment:
You must have at least one line of text between levels of headers.

Reply:
Done.